### DISCUSSION TITLE
請問cut scene 過場動畫裡的字幕如何翻譯， 有沒辦法用中文cut scene資料文件直接取代?

### DISCUSSION THREAD

**Discussion by ozonosphere** [GpointChen/FFXIVChnTextPatch-GP#388] [General]

**Comment by GpointChen** [comment]
resource/rawexd/cut_scene 應該就有過場動畫的翻譯。

不確定是否有部分動畫不給翻譯，你可以貼看看沒有翻譯的範例嗎？

**Comment by ozonosphere** [comment]
在遊戲裡The Unending Journey （旅行筆錄） 重生之境主線任務， 15級劇情，過場動畫4， 裡面有飛船 還有語音，但是字幕不是中文的。

**Reply by GpointChen** [reply]
這一段可能不給改，雖然在原文的quest\005\ManFst200_00507.csv裡有找到相同的文本，但並不完整，很可能是寫死的。

**Reply by ozonosphere** [reply]
好的，謝謝
